### ISSUE TITLE
Remove print statements when using wolframscript

### ISSUE THREAD

**Issue by og113** [Wall-Go/WallGoMatrix#32]
From Lauri's comment:
- Fixing the installation directory structure I can now run eg. wolframscript -file qcd.m in no-GUI mode. On Windows it runs happily but does a lot of unnecessary printing (authors, Grid[{{-Image, ...] etc. On Ubuntu it also runs but the prints are absolutely horrible, for example the images are printed in what looks like arrays of RGBA values. So I strongly propose all unnecessary prints are removed if running without the Mathematica frontend.

**Comment by pschicho** [comment]
Fixed on latest PR.

One error message remains which is actually an issue of `DRalgo` so not to be fixed here:
```
SetDelayed::write: Tag BlockDiagonalMatrix in BlockDiagonalMatrix[blocks_] is Protected.
```

**Comment by pschicho** [comment]
> Fixed on latest PR.
> 
> One error message remains which is actually an issue of `DRalgo` so not to be fixed here:
> 
> ```
> SetDelayed::write: Tag BlockDiagonalMatrix in BlockDiagonalMatrix[blocks_] is Protected.
> ```

Correction: this error is actually a `GroupMath` error. Therefore nothing we can do here.

**Comment by pschicho** [comment]
Another correction: the error was temporarily fixed by unprotecting `BlockDiagonalMatrix` upon GroupMath load. This is not ideal but GroupMath just overrides a Mathematica command and this issue has not been fixed since v1.1.2 from 2020.
